main.go

```diff
@@ -115,25 +126,32 @@ func realMain(args []string) int {
 			sem <- 0
 			defer func() { <-sem }()
 
-			loader := gojsonschema.NewReferenceLoader(fileUri(path))
-			result, err := schema.Validate(loader)
-			switch {
-			case err != nil:
-				msg := fmt.Sprintf("%s: error: %s", path, err)
+
+			loader, err := jsonLoader(path)
+			if err != nil {
+				msg := fmt.Sprintf("%s: unable to load doc: %s\n", *schemaFlag, err)
 				fmt.Println(msg)
 				errors = append(errors, msg)
```

Comment:
Can you add a `continue` here and remove the `else` clause to reduce the nesting one level.

Reply:
yep. great suggestion.. it felt icky when I put that else in, but didn't have a better idea. done.

Reply:
oh.. continue no workie.. because even tho it is inside a for loop, that is inside a go routine and it doesn't know.. 

i suppose I can do a return.

Reply:
done